CONTRIBUTING.md

```diff
@@ -92,6 +92,7 @@ When you're sending a pull request:
 
 - Prefer small pull requests focused on one change.
 - Verify that linters and tests are passing.
+- Run `npx expo-doctor` and make sure that your changes don't break any of it's checks.
```

Comment:
At some point it might be worth to add this as CI check?

Reply:
Agreed. I have created FCE-959 for that